src/driver/amdxdna/amdxdna_gem.c

```diff
@@ -21,7 +21,7 @@
 
 #define XDNA_MAX_CMD_BO_SIZE	SZ_32K
 
-MODULE_IMPORT_NS(DMA_BUF);
+MODULE_IMPORT_NS("DMA_BUF");
```

Comment:
This will fail for compiling with latest linux kernel. Just wondering what linux kernel are you build with? And what error did you see?

Reply:
I actually compiled this against Linux 6.13rc2,  the absolute latest available kernel version.

Reply:
Ok, you are right. It was DMA_BUF with rc1 and changed to "DMA_BUF" in rc2. In this case, please add something like:
#if LINUX_VERSION_CODE < KERNEL_VERSION(6, 13, 0)
MODULE_IMPORT_NS(DMA_BUF)
#else 
MODULE_IMPORT_NS("DMA_BUF")
#endif

Reply:
@houlz0507 added :)